config.jsonc

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
since this is copied when deloying to gh pages it should  have the  final production config in it

Reply:
Yes, I am thinking we should put some example config up, and actual running dashboards can be repository forks with their own config + optionally theme changes.